spec/services/random_phrase_spec.rb

```diff
@@ -21,5 +21,13 @@
 
       expect(phrase.to_s.length).to eq 24 # 20 chars + 4 spaces
     end
+
+    it 'does not contain the letters I L O U' do
+      100.times do
```

Comment:
`100` is an arbitrary number reasonably big enough to "randomly" trigger the presence of one of the suspect numbers.

Maybe we should just remove this test, since we can rely on the Base32 algorithm to avoid those letters.